app/Services/UpdateProfileService.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brewmap\Services;
+
+class UpdateProfileService
+{
+    public function updateProfileData(array $input): void
+    {
+        auth()->user()->profile->update($input);
```

Comment:
Here, you can also try to use `Illuminate\Contracts\Auth\Guard`
```suggestion
        $this->guard->user()->profile->update($input);
```